app/views/tasks/show.html.slim

```diff
@@ -535,6 +552,21 @@
           div data-bs-toggle='tooltip' title=unavailable_tooltip data-bs-delay=150
             = button_tag class: 'btn btn-outline-dark dropdown-toggle disabled', 'data-bs-toggle': 'dropdown' do
               = t('.button.export')
+  .col-md-4.col-sm-6.mb-3.p-0
+    .button-box.border.p-3.h-100
+      .button-description
+        = t('.contribution_list_hint')
+      .wrapper
+        - if !@task.contribution? && policy(@task).edit?
+          = link_to t('.button.contribution_list'), task_task_contributions_path(@task), class: 'btn btn-light', 'data-turbolinks' => false
+        - else
+          - if @task.contribution? || @current_user.nil?
+            - contrib_list_unavailable_tooltip = unavailable_tooltip
+          - else
+            - contrib_list_unavailable_tooltip = t('.owner_required_tooltip')
+          div data-bs-toggle='tooltip' title=contrib_list_unavailable_tooltip data-bs-delay=150
+            = link_to t('.button.contribution_list'), task_task_contributions_path(@task), class: 'btn btn-outline-dark disabled', 'data-turbolinks' => false
```

Comment:
I would assume you don't need the `'data-turbolinks' => false`, do you? This is only added for some frontend libraries (like select2), but since the target page doesn't (yet) contain such parts, we don't need to disable it.

Reply:
The data attribute was removed.